app/src/components/TransactionStatus.vue

```diff
@@ -1,53 +1,50 @@
 <template>
   <!-- Transaction hash -->
-  <div :class="[...rowClasses, 'border-b', 'border-grey-100']">
-    <div class="col-span-12 md:col-span-3 grid-rows-3">Hash:</div>
-    <div class="col-span-10 md:col-span-6">
-      <a :href="etherscanUrl" target="_blank" rel="noopener noreferrer" class="link text-grey-400">{{ hash }}</a>
+  <section>
+    <label>Hash:</label>
+    <div class="">
+      <a :href="etherscanUrl" target="_blank" rel="noopener noreferrer " class="link break-all">{{ hash }}</a>
```

Comment:
Nit, extra space here

```suggestion
      <a :href="etherscanUrl" target="_blank" rel="noopener noreferrer" class="link break-all">{{ hash }}</a>
```